scripts/makefile/backup.mk

```diff
@@ -6,6 +6,6 @@ backup_name = $(COMPOSE_PROJECT_NAME)-$(datestamp).tar.gz
 
 backup:
 	rm -f $(backup_name)
-	$(call php, drush sql-dump --database=default --result-file=../$(mysql_dump_name))
+	$(call php, drush sql-dump --database=default --result-file=../$(mysql_dump_name) --structure-tables-list=cachetags,cache_*,flood,sessions,watchdog --extra=--single-transaction)
```

Comment:
no need for -extra=--single-transaction
drush will add it automatically

Reply:
Please create followup pr